community/vmwh/build

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh -e
+
+# Remove dependency on bsd make.
+patch -p1 < makefile.patch
+
+# Include stdint.h to fix compile error.
+patch -p1 < stdint.patch
+
+# Add missing __packed macro.
+patch -p1 < packed-attr.patch
+
+# Remove debugging code that needs bsd strnvis(3).
+patch -p1 < remove-vis-dep.patch
```

Comment:
```sh
for patch in *.patch; do
    patch -p1 < "$patch"
done
```

Reply:
I just amended the pull request to include this.